### DISCUSSION TITLE
How to run backend individually

### DISCUSSION THREAD

**Discussion by GaganUnvi** [eclipse-theia/theia#8693] [General]
How to use a backend services without front-end UI interaction like only backend is running and you could send a request from postman to receive response in something json format ?

**Comment by zhaomenghuan** [comment]
@GaganUnvi Hello，what are the scenarios that do not require a front-end interface? If you don’t need a front-end, why don’t you build a service directly based on Node.js.

**Reply by GaganUnvi** [reply]
@zhaomenghuan Hi, no I do need front-end interface the thing is that we are trying build an VR IDE using various APIs like WebXR and we also have created a text editor for the same and now we want to integrate that editor with theia's backend but the frontend and backend seems to be interdependent with each other in the examples given in your repos.
